packages/ui/src/lib/layouts/Page.svelte

```diff
@@ -97,7 +116,10 @@ function handleKeydown(e: KeyboardEvent): void {
   {#if inProgress}
     <LinearProgress />
   {/if}
-  <div class="flex flex-row px-2 border-b border-[var(--pd-content-divider)]" aria-label="Tabs" role="region">
+  <div
+    class="flex flex-row px-2 {`pt-${heightOfDetail > 50 ? 4 : 0}`} border-b border-[var(--pd-content-divider)]"
```

Comment:
This would not scale very well as tailwind classes are generated at compile time

![image](https://github.com/user-attachments/assets/f2b1cd05-da96-4727-ba06-a403590aac0d)

(ref https://v2.tailwindcss.com/docs/just-in-time-mode)

So you need to use the style here instead of class

Reply:
and probably better to use height property instead of padding?